docs/docs/cmd/purview/retentionevent/retentionevent-add.md

```diff
@@ -0,0 +1,146 @@
+# purview retentionevent add
+
+Create a retention event
+
+## Usage
+
+```sh
+m365 purview retentionevent add [options]
+```
+
+## Options
+
+`-n, --displayName <displayName>`
+: The display name of the event
+
+`-t, --eventType <eventType>`
```

Comment:
I think you missed reworking this PR, according to our conversation [here](https://github.com/pnp/cli-microsoft365/issues/4373#issuecomment-1439407760) and the updated specs.

Reply:
Correct! Sorry about that! I've used an in code filter method for retrieving the event types by name since the filter property is not accepted by the graph url

Reply:
Thanks ok!